website/content/docs/connect/config-entries/proxy-defaults.mdx

```diff
@@ -153,415 +259,488 @@ spec:
+
+- [Envoy proxy configuration option](/consul/docs/connect/proxies/envoy#proxy-config-options)
+- [Built-in proxy configuration options](/consul/docs/connect/proxies/built-in#proxy-config-key-reference) 
+
+#### Values
+
+- Default: None
+- Data type: Map 
+
+### `spec.envoyExtensions`
+
+Specifies a list of extensions that modify Envoy proxy configurations. Refer to [Envoy extensions](/consul/docs/connect/proxies/envoy-extensions) for additional information.
+
+#### Values
+
+- Default: None
+- Data type: Map of containing the following fields:
+   - `name`
+   - `required`
+   - `arguments`
+   - `consulVersion`
+   - `envoyVersion`
+
+The following table describes how to configure values in the `EnvoyExtensions` map:
+
+| Parameter | Description | Data type | Default |
+| ---       | ---         | ---       | ---     |
+| `name` | Specifies the name of the extension. | String | None |
+| `required` | Specify `true` to require Consul to implement the extension. <p>Use this parameter to ensure that extensions required for secure communication are not unintentionally bypassed.</p><p>Consul returns an error when `Required` is set to `true` and the extension does not update Envoy resources. </p> | String | None |
+| `arguments` | Specifies the arguments to pass to the extension executable. Refer to the documentation for the extension you want to implement for additional information. | Map | None |
+| `consulVersion` | Specifies the version of Consul that the extension is allowed to work with. Consul validates the version during xDS updates. If a different version is in use, Consul skips the extension and writes the event to the log. <p>The `consulVersion` and `envoyVersion` must both validate for Consul to implement the extension.</p> | String | None |
+| `envoyVersion` | Specifies the version of Envoy that the extension is allowed to work with. Consul validates the version during xDS updates. If a different version is in use, Consul skips the extension and writes the event to the log. <p>The `consulVersion` and `envoyVersion` must both validate for Consul to implement the extension.</p> | String | None |
+
+### `spec.mode`
+
+Specifies a mode for how proxies direct inbound and outbound traffic. You can specify one of the following values:
 
-</CodeTabs>
+- `transparent`: In transparent mode, proxies capture and redirect inbound and outbound traffic. The mode does not enable traffic redirection, but directs Consul to configure Envoy as if traffic is already being redirected. 
+
+- `direct`: In this mode, the local application and other proxies must directly dial proxy listeners. 
+
+#### Values
+
+- Default: None
+- Data type: String 
+
+### `spec.transparentProxy`
+
+Contains configurations for proxies that are running in transparent proxy mode. Refer to [Transparent proxy mode](/consul/docs/k8s/connect/transparent-proxy) for additional information. 
```

Comment:
```suggestion
Contains configurations for proxies that are running in transparent proxy mode. This mode enables permissive mTLS for Consul so that you can use your Kubernetes cluster's DNS service instead of Consul DNS. Refer to [Transparent proxy mode](/consul/docs/k8s/connect/transparent-proxy) for additional information. 
```